### PR TITLE
Added ensino folder

### PR DESCRIPTION
**Pull request by Mike-Sundays**

### CODE REVIEW

File: portugues_pt/ensino/alternativo.md [hunk 1]
```diff
@@ -0,0 +1 @@
+Lutar por um sistema de ensino alternativo, que reconhecesse as qualidades únicas de cada ser humano.
```
Comment (casey-collab) [inline portugues_pt/ensino/alternativo.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

**Comment by Mike-Sundays** [comment]
Please review this, @casey-collab". Make sure to leave the comment on the "Conversation" tab of the pull request, not the "Files changed

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

File: portugues_pt/colaborar.md [hunk 2]
```diff
@@ -14,6 +14,8 @@ Contribuía para [obras de caridade](caridade/caridade.md)
 
 Compraria [componentes eletronicos](componentes/componentes.md)
 
+Revolucionava o sistema de [ensino](ensino/alternativa.md)
```
Comment (casey-collab) [inline portugues_pt/colaborar.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

File: portugues_pt/ensino/alternativo.md (hunk 1 above)
Comment (casey-collab) [inline portugues_pt/ensino/alternativo.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

**Comment by Mike-Sundays** [comment]
Please review this, @casey-collab

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by Mike-Sundays** [comment]
Please review this, @casey-collab

**Comment by casey-collab** [comment]
Looks great! :sunny:

**Pull request merged**